frontend/src/constants/localStorage.ts

```diff
@@ -18,5 +18,5 @@ export enum LOCALSTORAGE {
 	DASHBOARD_VARIABLES = 'DASHBOARD_VARIABLES',
 	SHOW_EXPLORER_TOOLBAR = 'SHOW_EXPLORER_TOOLBAR',
 	PINNED_ATTRIBUTES = 'PINNED_ATTRIBUTES',
-	THEME_ANALYTICS = 'THEME_ANALYTICS',
+	THEME_ANALYTICS_V1 = 'THEME_ANALYTICS_V1',
```

Comment:
renamed the local storage key as older keys would have already been set for the user.